app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SessionsController < ApplicationController
+  skip_before_filter :require_login, except: [:destroy]
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    if @user = login(session_params[:email], session_params[:password])
+      redirect_to review_path, notice: "Logged in!"
+    else
+      flash.now[:alert] = 'Login failed'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.